modules/storages/spec/factories/storage_factory.rb

```diff
@@ -144,8 +144,10 @@
                                       "MISSING_NEXTCLOUD_LOCAL_OAUTH_CLIENT_REFRESH_TOKEN"),
              token_type: "bearer")
 
-      create(:remote_identity, oauth_client: storage.oauth_client, user: evaluator.oauth_client_token_user,
```

Comment:
🟡 If you already did a small cleanup here, maybe you can quickly shoot at line number 96? Replace it with `automatic_management_enabled` to remove the deprecated method there?

Reply:
got it.